packages/core/src/remote_operation/type.ts

```diff
@@ -147,4 +147,5 @@ export type DataSource<Params> = {
   >;
   set?: Effect<{ params: Params; result: unknown }, void, unknown>;
   unset?: Effect<{ params: Params }, void, unknown>;
+  $enabled: Store<boolean>;
```

Comment:
Should it be optional? Or it's ok to add new required parameter bcs its `lowLevelAPI` ?

Reply:
I would prefer to make it optional because now it sounds like an unnecessary breaking change.